src/ServiceControl.Monitoring/Http/Diagrams/DiagramApiModule.cs

```diff
@@ -107,8 +148,21 @@ static MonitoredEndpoint[] GetMonitoredEndpoints(EndpointRegistry endpointRegist
                 {
                     Name = endpoint.Key,
                     EndpointInstanceIds = endpoint.Value.Select(i => i.InstanceId).ToArray(),
-                    IsStale = endpoint.Value.Any(i => activityTracker.IsStale(i))
+                    IsStale = endpoint.Value.Any(activityTracker.IsStale)
                 }).ToArray();
         }
+
+        static MonitoredEndpointMessageType[] GetMonitoredMessageTypes(IEnumerable<IProvideBreakdownBy<EndpointMessageType>> stores, HistoryPeriod period, string endpointName)
```

Comment:
Maybe we should a monitored message type registry in the same way that we have an endpoint registry?